Formula/solana.rb

```diff
@@ -0,0 +1,43 @@
+class Solana < Formula
+  desc "Web-Scale Blockchain for decentralized apps and marketplaces"
+  homepage "https://solana.com"
+  url "https://github.com/solana-labs/solana/archive/refs/tags/v1.7.11.tar.gz"
+  sha256 "ed70aaed16910eab637eff0499c6e4762350c4ad50340fcb1b4bf94d23f1f97a"
+  license "Apache-2.0"
+
+  depends_on "protobuf" => :build
+  depends_on "rust" => :build
+
+  uses_from_macos "llvm" => :build
```

Comment:
Last question, and I think this is ready to merge: what does this use LLVM for? If it's needed for the `clang`, then I think we can drop this as well, since `brew` sets `CC=gcc-5` and `CXX=g++-5` on Linux to use the system compiler by default.

Reply:
@carlocab Hmm, good spot. According to Solana's docs: https://github.com/solana-labs/solana#building `llvm` and `clang` are required on Linux. I'm pretty sure this is used for C FFI modules. Not sure if that has anything to do with the fact that `rustc` also uses LLVM. I'll re-run the build on Linux without the LLVM macro.

Reply:
Removed LLVM dependency without issues.